gammapy/datasets/simulate.py

```diff
@@ -532,11 +532,11 @@ def event_list_meta(dataset, observation, keep_mc_id=True):
         if loc is None:
             if telescope == "CTA":
```

Comment:
For the initial release, the stored metadata is really CTA?
For the next release, it might be CTAO. So I propose to add an OR condition with telescope == "CTAO"

To @morcuended and Matthias , should we keep in mind that the CTAO data releases will use the following metadata : 
- TELESCOP: "CTAO"
- INSTRUME: "Southern Array" and "Northern Array"
Is it the anticipated standard values?